libcontainer/intelrdt/cmt_test.go

```diff
@@ -13,10 +13,10 @@ func TestGetCMTNumaNodeStats(t *testing.T) {
 		"llc_occupancy": 9123911,
 	}
 
-	mockedL3_MON, err := mockResctrlL3_MON(mocksNUMANodesToCreate, mocksFilesToCreate)
```

Comment:
I think it's called L3_MON because there is a directory with the same name under /sys/fs/resctrl (`L3_MON`), and thus rename will make things harder to find. Not super important of course but I'd rather keep this as is.

https://github.com/opencontainers/runc/blob/3a041e9654169a301c19cfcaa52df2dc16115db2/libcontainer/intelrdt/intelrdt.go#L63